src/status_im/translations/en.cljs

```diff
@@ -25,6 +25,8 @@
    :add                                  "Add"
    :yes                                  "Yes"
    :no                                   "No"
+   :on                                   "On"
+   :off                                  "Off"
```

Comment:
Not sure you're using on/off or yes/no in toggle switcher

Reply:
What do you mean? It's On/Off in mockups

Reply:
I mean it's "Yes/No" right now in UI
```
:value               (i18n/label (if sharing-usage-data? :yes :no))
```

Reply:
Oups